stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/factory/FileAttachmentFactory.kt

```diff
@@ -15,7 +16,11 @@ import io.getstream.chat.android.compose.ui.theme.ChatTheme
  */
 @Suppress("FunctionName")
 public fun FileAttachmentFactory(): AttachmentFactory = AttachmentFactory(
-    canHandle = { attachments -> attachments.any { it.uploadId != null || it.upload != null } },
+    canHandle = { attachments ->
+        attachments.any {
+            it.uploadId != null || it.upload != null || it.type == ModelType.attach_file || it.type == ModelType.attach_video || it.type == ModelType.attach_video
```

Comment:
The formatting on this is supper ugly and I am open to suggestions as personally I cannot find a way to make it prettier.

Reply:
Also the reason why we do this is because at least in one case we have received a video attachment containing no mime type and a file type.

Reply:
Why do we check that `it.type == ModelType.attach_video` twice?

Reply:
That's a mistake that, well spotted.